src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -79,6 +84,30 @@ public function setBuilder(NodeBuilder $builder): void
         $this->nodeBuilder = $builder;
     }
 
+    /**
+     * Allows alternative types and wraps them into arrays.
+     *
+     * @param list<ExprBuilder::TYPE_*> $allowedTypes The type of values that should be wrapped in an array
+     *
+     * @return $this
+     */
+    public function acceptAndWrap(array $allowedTypes, ?string $key = null): static
+    {
+        $this->allowedTypes = $allowedTypes;
+
+        foreach ($allowedTypes as $type) {
+            $this->beforeNormalization()->ifTrue(match ($type) {
+                ExprBuilder::TYPE_INT => is_int(...),
+                ExprBuilder::TYPE_STRING => is_string(...),
+                ExprBuilder::TYPE_BOOL => is_bool(...),
+                ExprBuilder::TYPE_NULL => is_null(...),
+                ExprBuilder::TYPE_BACKED_ENUM => static fn ($v) => $v instanceof \BackedEnum,
```

Comment:
This does not handle `ExprBuilder::TYPE_ANY`, while this is a valid value in the documented type `list<ExprBuilder::TYPE_*>`

Reply:
Yes, on purpose, but this means I should list only the allowed subset, right?